keras_cv/layers/preprocessing/equalization.py

```diff
@@ -60,9 +60,15 @@ def equalize_channel(self, image, channel_index):
         histogram = tf.histogram_fixed_width(image, [0, 255], nbins=self.bins)
 
         # For the purposes of computing the step, filter out the nonzeros.
-        nonzero = tf.where(tf.not_equal(histogram, 0))
-        nonzero_histogram = tf.reshape(tf.gather(histogram, nonzero), [-1])
-        step = (tf.reduce_sum(nonzero_histogram) - nonzero_histogram[-1]) // (
+        # Zeroes are replaced by 1410065408 while calculating min to keep shape
+        # constant across input sizes for compatibility with vectorized_map
+        histogram_without_zeroes = tf.where(
+            tf.equal(histogram, 0),
+            tf.multiply(tf.ones((), tf.int32), 1410065408),
```

Comment:
Can we add a tiny test case asserting that 1410065408 shows up nowhere in the result?  Just pass a random uniform distribution with value_range=(0, 255) and then assert 1410065408 doesn't show up,

Reply:
Done! I added a more general testcase that ensures that the resultant output range is the same. This will exclude 1410065408 as well.

However, 1410065408 is not really interfering with the range and simply changing the frequency.